mne/viz/topo.py

```diff
@@ -489,9 +754,25 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0.3, vmin=None,
     border : str
         matplotlib borders style to be used for each sensor plot.
     fig_facecolor : str | obj
-        The figure face color. Defaults to black.
+        The figure face color. Defaults to black. If "black", spines are
+        white (else they are black).
     font_color : str | obj
-        The color of tick labels in the colorbar. Defaults to white.
+        The color of text in the colorbar and title. Defaults to spine_color
```

Comment:
I'd prefer to allow at least the full range of monochrome options, as people plot a lot on grey backgrounds these days.
For that range, selecting spine and text color is also simple - depending on if face color is closer to white or to black, the others are set to black or white, respectively.